semcore/dropdown-menu/src/DropdownMenu.jsx

```diff
@@ -146,7 +188,7 @@ class DropdownMenuRoot extends AbstractDropdown {
         (e.key === 'ArrowLeft' && placement?.startsWith('right')) ||
         (e.key === 'ArrowRight' && placement?.startsWith('left')) ||
         e.key === 'Escape';
-      const isMenuItem = e.target.getAttribute('role') === super.childRole;
+      const isMenuItem = e.target.getAttribute('role')?.startsWith(super.childRole);
```

Comment:
Why `startsWith`? Shouldn't it be exact match?

Reply:
no, because here we compare `menuitem` and `menuitemradio` for example